src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -2210,7 +2210,7 @@ public async Task<JArray> GetArrayValues(SessionId sessionId, int arrayId, Cance
             }
             return array;
         }
-        public async Task<bool> EnableExceptions(SessionId sessionId, string state, CancellationToken token)
+        public async Task<bool> EnableExceptions(SessionId sessionId, PauseOnExceptionsKind state, CancellationToken token)
```

Comment:
nit: might be useful to add a check for `Unset` here anyway, to guard against accidentally calling this with `Unset`, and log a debug message?

Reply:
Done.